content/client-server-api/modules/content_repo.md

```diff
@@ -23,19 +23,67 @@ When serving content, the server SHOULD provide a
 interacting with the media repository.
 {{% /boxes/added-in-paragraph %}}
 
+{{% boxes/added-in-paragraph %}}
+{{< changed-in v="1.11" >}} The unauthenticated download endpoints have been
+deprecated in favour of newer, authenticated, ones. This change included updating
```

Comment:
Nit, but mixing tenses here: probably just, "This change includes"?. Also... 'pathing'? Is that a word? Maybe just 'paths'?